### PR TITLE
[ISSUE #3492] Fixed action text style

### LINKED ISSUE #3492: "Next" button in group chat screen is too small
### Description
*Type*: Bug

*Summary*: Size of "Next" button in group chat screen is small. In result, it maybe hard to click it. 

#### Expected behavior
"Next" button have the same size as "Create":

#### Actual behavior
"Next" button size is really small. It's hard to tap:

 

### Reproduction
- Open Status
- Go to Home
- Create group chat
- Select members for group chat

### Additional Information
* Status version: nightly, March 1
* Operating System: iOS only
* Tested on iPhone 6, iOS 11.2.1

### PR DESCRIPTION
**Pull request by jeluard**
fixes #3492

### Summary:

Improved toolbar action text style

### Steps to test:
- Open Status
- Create a group chat
- Make sure toolbar styles match mockups

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/group/add_contacts/views.cljs [hunk 1]
```diff
@@ -70,20 +67,6 @@
       (:name group)]
      [toggle-list contacts group-toggle-contact]]))
 
-(defn toggle-participants-handler []
```
Comment (jeluard) [inline src/status_im/ui/screens/group/add_contacts/views.cljs:73]: This was unused

File: src/status_im/ui/screens/group/add_contacts/views.cljs [hunk 2]
```diff
@@ -70,20 +67,6 @@
...
-  (re-frame/dispatch [:add-new-group-chat-participants])
-  (re-frame/dispatch [:navigate-back]))
-
-(defn add-participants-toggle-list-toolbar [selected-contacts-count]
```
Comment (jeluard) [inline src/status_im/ui/screens/group/add_contacts/views.cljs:77]: This was unused

File: src/status_im/ui/screens/group/views.cljs [hunk 3]
```diff
@@ -13,15 +13,6 @@
             [status-im.ui.screens.group.styles :as styles]
             [status-im.ui.screens.group.db :as v]))
 
-(defn group-toolbar [group-type edit?]
```
Comment (jeluard) [inline src/status_im/ui/screens/group/views.cljs:16]: This was unused

**Review by alwx — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]

**Comment by asemiankevich** [comment]
@jeluard is that expected? Develop looks good to me

image

**Comment by jeluard** [comment]
@asemiankevich Should be fixed now.

**Comment by lukaszfryc** [comment]
@jeluard the font is indeed bigger than described in the issue #3492. But, maybe we could increase it even more to match the font of "Done" button while editing profile. Also, let's make sure the font has the same size in every other place (unify it).  

**Font of "Next" is still a bit small** 



**"Done" seems just right :)**



@denis-sharypin what do you suggest?

**Comment by jeluard** [comment]
@lukaszfryc Font matches latest mockups. If it's different from the other one then it will need some changes. This will happen gradually while we are moving to more components usage.

**Pull request merged**